chroma_feedback/consumer/razer_chroma/device.py

```diff
@@ -50,7 +58,7 @@ def process_devices(devices : Any, status : str) -> List[Dict[str, Any]]:
 				'consumer': 'razer_chroma',
 				'type': 'device',
 				'name': device.name,
-				'active': pulsate_device(device, color.get_failed()),
+				'active': globals()[effect_method](device, color.get_failed()),
```

Comment:
Same as above... that `globals()[effect_method]` thing is an anti-pattern by the way

Reply:
It may well be; my Python knowledge is somewhat uneven and I'm not sure what the 'right' way to do the function pointer would be.  Combining your two comments, you're saying you want:
```
'active': pulsate_device(device, color.get_failed()) or breath_device(device, color.get_errored())
```
I think I would go ahead and make a catch-all 'strobe_device()' call to perform both of these calls (maybe strobe is the wrong terminology to use; I need to look at the other effects again). 

My only concern is that I *think* both these calls return True regardless of whether or not they work or not.  I guess it's not *really* a problem as long as the device lights up...?

Reply:
This is would be an option, but as of clean coding it is better to have two methods that describe itself by their name. If you have a strobe_device() method that does conditional actions you have to read the whole code block to understand it.